apps/hash-ai-worker-ts/scripts/compare-llm-response.ts

```diff
@@ -144,6 +88,7 @@ export const compareLlmResponses = async () => {
   const llmResponses = await Promise.all(
     models.map((model) => {
       return getLlmResponse(
+        // @ts-expect-error - figure out what's going wrong here
```

Comment:
The `models` field accepts any model name, but the params demands that only OpenAI model names are sent with OpenAI params, etc.

The type of `CompareLlmResponseConfig` would have to be updated somehow to make sure they were in sync, maybe with generics or something. Not sure it's worth sinking a lot of time into, since it's a testing utility.

Reply:
Yeah, I made an attempt at this but didn't think it was worth sinking a lot of time into. We'll probably revise this method further once we dedicate time on the model evaluation you've mentioned.